components/ArtifactChips.vue

```diff
@@ -168,10 +172,17 @@ export default {
             return '/artifact/' + item.related_artifact_group_id
           case 'reverse-relation':
             return '/artifact/' + item.artifact_group_id
+
         }
       }
       return null
     },
+    navigateTo(item){
```

Comment:
This seems like an odd name for the function, but maybe it 's just me.  navigateTo seems like a function that would move the scroll bar to a particular element but this is really navigating.  I suppose it's returning a navigation link though.  Maybe call it `labelURL` instead?  Your call though.